Core/GDCore/Project/LoadingScreen.cpp

```diff
@@ -11,7 +11,7 @@
 namespace gd {
 
 LoadingScreen::LoadingScreen()
-    : showGDevelopSplash(true),
+    : showGDevelopSplash(false),
```

Comment:
why change this?

Reply:
So that a new project has the watermark enabled by default, and not the loading screen. Should make both branding options enabled by default?

Reply:
Let's try the watermark only for now.